### ISSUE TITLE
NONMEM 7.5 issue with add_iiv

### ISSUE THREAD

**Issue by SCarter79** [pharmpy/pharmpy#112]
Pharmpy 0.17.0 running Python 3.8.7 through RStudio 1.3.1056 and R 4.0.3. I added an iiv to a model, but this is not recognised in NONMEM

```
pharmpy$plugins$nonmem$conf$parameter_names <- c('abbr', 'comment', 'basic')
model1 <- pharmpy$Model(here::here('run1.mod'))
model6 <- model1$copy() %>%
  modeling$update_inits(force_individual_estimates = TRUE) %>%
  modeling$add_peripheral_compartment() %>%
  modeling$add_iiv('QP1', expression = 'exp', operation = '*') %>%
  modeling$add_iiv('VP1', expression = 'exp', operation = '*') %>%
  modeling$fix_parameters(c('IIV_QP1','IIV_VP1')) %>%
  modeling$update_source() 
  
model6$parameters$inits['IIV_QP1'] = 0
model6$parameters$inits['IIV_VP1'] = 0
modeling$update_source(model6)  

modeling$write_model(model6, here::here('run6.mod'), force = TRUE)

```

gave the following error:

![image](https://user-images.githubusercontent.com/40889227/108093631-31492700-707e-11eb-94ff-e06ae8435cc0.png)

**Comment by stellabelin** [comment]
Thanks for reporting this! I can reproduce it, it seems the new ETAs are not being replaced with their NONMEM name.

**Comment by stellabelin** [comment]
Found that this bug does no longer appear, while using 0.17.0 still causes this issue, should be fixed for 0.18.0.
